app.js

```diff
@@ -0,0 +1,8 @@
+var portfolioArray = [];
```

Comment:
ensure that you are calling 'use strict' in every js file. This will save you big time once we start doing more with JavaScript.